examples/python/spi_loopback.py

```diff
@@ -21,20 +21,23 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE
+#
+# Example Usage: Loopbacks data between MISO and MOSI 100 times
 
-# Read from MCP3004 ADC pin 0 in single ended mode
-import mraa
-import time
+import mraa as m
+import random as rand
 
-dev = mraa.Spi(0)
-txbuf = bytearray(3)
-txbuf[0] = 0x01
-txbuf[1] = 0x80
-txbuf[2] = 0x00
+# intialise SPI
+dev = m.Spi(0)
 
-while True:
-    rxbuf = dev.write(txbuf)
-    value = ((rxbuf[1] & 0x03) << 8) | rxbuf[2]
-    print value
-    time.sleep(0.5)
+for x in range(0,100):
+    txbuf = bytearray(4)
+    for y in range(0,4):
+        txbuf[y] = rand.randrange(0, 256)
 
+    # send and receive data through SPI
+    rxbuf = dev.write(txbuf)
+    if rxbuf != txbuf:
+        print("Data mismatch!")
+        break
+        exit(1)
```

Comment:
Unreachable code. I know it was in the original example, but let's just fix that while we are at it. I'd suggest removing the `break` and leaving just the `exit(1)`, that would be in line with my fix for other example.

Reply:
Ack